src/translations/en.json

```diff
@@ -1163,7 +1163,7 @@
             },
             "reloading": {
               "heading": "YAML configuration reloading",
-              "introduction": "Some parts of Home Assistant can reload without requiring a restart. Hitting reload will unload their current YAML configuration and load the new one.",
+              "introduction": "Some parts of Home Assistant can reload without requiring a restart. Clicking the appropriate Integration below will unload their current YAML configuration and load the new one.",
```

Comment:
They are not necessarily integrations, things like core config and customization, etc., are also in there.

Reply:
check, thanks

Reply:
btw, core config isn't listed, thought that service still exists

Reply:
It's listed for me 
![image](https://user-images.githubusercontent.com/15093472/124917615-52da8900-dff4-11eb-96e3-9a07b305c63e.png)

Reply:
yes, we have that, but it doesn't mention Core config, like it used before.

This is a 'have to know, or otherwise no way to deduce from the Frontend info' thing. But, no matter, I changed the word, as per Bram's comment

Reply:
I indeed lack info now, as more things have been added to core config. That could be reworded too.